kukur/source/piwebapi_af/piwebapi_af.py

```diff
@@ -145,6 +147,10 @@ def _read_data(
                 timeout=self.__request_properties.timeout_seconds,
                 params=params,
             )
+            if response.status_code == NOT_FOUND:
+                logger.warning(f"No data found for {data_url}")
```

Comment:
For logging, it's recommended to use `%s` instead of f-strings as the `%s` is not evaluated when the logging level is not active.

We used to have a lint for this.